indexer/app/lib/indexer_common.rb

```diff
@@ -158,6 +158,16 @@ def self.build_fullrecord(record)
     fullrecord
   end
 
+  # There's a problem with how translation paths get loaded when selenium tests are run
+  # Call this instead of I18n.t so that it won't cause an error when selenium tests are run
+  @@selenium = Dir.getwd.end_with? 'selenium'
+  def t(*args)
+    if @@selenium
+      args[0]
+    else
+      I18n.t(*args)
```

Comment:
@sem42198 Thinking about the comment I left last night, could this be modified in some fashion to catch translation not found issues and, instead, just output the key instead of a translation error?

So, instead of storing: `translation missing: en.reports.created_accessions.title` as the title field in the SOLR index, it would just store `created_accessions`.

Reply:
That is one possibility. I18n also allows for passing in a default value which would have the same effect. Regardless, I don't think there is much that can be done about not having the translations for those report types aside from just outputting the key since as far as I know it is the responsibility of the report to provide that translation and would probably have additional implications beyond this if the report does not provide a title translation.

Reply:
Can you make it so that there is a default value of the untranslated key in that case?  I don't care much about the custom job types (indeed, that is an issue for the plugin creator), but since a lot of this translation missing errors are hanging around because of the refactor that happened to core-shipped reports I feel like it is a bit of a regression to introduce translation errors into the index because of holdovers of past core-created values.